main.go

```diff
@@ -519,6 +520,15 @@ func setupWebhooks(mgr ctrl.Manager) {
 		setupLog.Error(err, "unable to create webhook", "webhook", "ExtensionConfig")
 		os.Exit(1)
 	}
+
+	if err := (&expwebhooks.IPAddress{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
The client has to be set here. Otherwise => nil pointer dereference

Reply:
Can you maybe do a few manual smoke tests with tilt? Just checking if the webhooks are called and ~ work?

Reply:
Tests done. RBAC also works.